diagnostic.md

```diff
@@ -5,6 +5,12 @@ GitHub repo.
 
 <!-- your answer starts here -->
 
+<!--#<body > div:nth-child(6)</body>
+ <div.modal-backdrop.js-touch-events
+<div id="start-of-content" class="accessability-aid<</div>
+<body > div.container.site-footer-container></body>
+-->
+
```

Comment:
These aren't quite right.  We want them in selector form, such as `body`, `div:nth-child(6)`, or `.container`.